internal/ui/progress/counter.go

```diff
@@ -66,13 +76,17 @@ func (c *Counter) Done() {
 	*c = Counter{} // Prevent reuse.
 }
 
-func (c *Counter) get() uint64 { return atomic.LoadUint64(&c.value) }
+// Get the current Counter value. This method is concurrency-safe.
+func (c *Counter) Get() uint64 { return atomic.LoadUint64(&c.value) }
+
+// GetMax returns the expected maximum. This method is concurrency-safe.
+func (c *Counter) GetMax() uint64 { return atomic.LoadUint64(&c.max) }
```

Comment:
This doesn't seem to get used outside the package.

Reply:
`getMax` is now unexported again. Looks like I've forgotten to clean it up along with the broken test code that used that method.

`Get` is still exported as it's used in testing that the progress bar behaves somewhat correctly. I wonder whether it would be better to add a testing helper method that makes it more obvious that `Get` should only be called by tests.